### PR TITLE
RF | OrganizationFolderTest 

### PR DESCRIPTION
**Pull request by asyniagina**
https://trello.com/c/ZnvQDkJv/1060-rf-classname-testname

### CODE REVIEW

**Comment by asyniagina** [comment]
[RF | OrganizationFolderTest > testCreateOrgFolder](https://trello.com/c/ZnvQDkJv/1060-rf-organizationfoldertest-testcreateorgfolder)

**Comment by asyniagina** [comment]
[RF | OrganizationFolderTest > testCreateOrgFolder](https://trello.com/c/ZnvQDkJv/1060-rf-organizationfoldertest-testcreateorgfolder)

**Comment by asyniagina** [comment]
https://trello.com/c/tMtvODuZ/1064-rf-organizationfoldertest-testorgfolderemptynameerr

File: src/test/java/tests/OrganizationFolderTest.java [hunk 1]
```diff
@@ -156,18 +159,23 @@ public void testCreateOrgFolder() {
...
-                        findElement(By.id("itemname-required")).getText(),
+        Assert.assertEquals(errMessageEmptyName,
                 "» This field cannot be empty, please enter a valid name");
-        Assert.assertFalse(getDriver().findElement(OK_BUTTON).isEnabled());
     }
 
+        //actualOrgFolderEmptyName
+        // .clickSaveButton()
+        // .getDisplayName();
+        //Assert.assertFalse(getDriver().findElement(OK_BUTTON).isEnabled());
```
Comment (UmidaKaharova) [inline src/test/java/tests/OrganizationFolderTest.java:177]: no comments please

Comment (asyniagina) [reply src/test/java/tests/OrganizationFolderTest.java]: comments deleted

**Review by SafronovaOlga — APPROVED** [review]

**Review by kartavenkopavel — APPROVED** [review]

**Pull request merged**